packages/apps/sales/src/components/atoms/SaleOrderLineCard/SaleOrderLineCard.tsx

```diff
@@ -0,0 +1,233 @@
+        }}
+        upperTexts={{
+          items: [
+            {
+              displayText: product.name,
+              isTitle: true,
+            },
+            {
+              displayText: productName,
+              hideIfNull: true,
+            },
+          ],
+        }}
+        lowerTexts={{
+          items: [
+            {
+              indicatorText: I18n.t('Sales_UnitPrice'),
+              displayText: formatPrice(price),
+              style: styles.noBold,
+            },
+            {
+              indicatorText: I18n.t('Sales_Unit'),
+              displayText: unit,
+              style: styles.noBold,
+            },
+          ],
+        }}
+        sideBadges={{
+          style: styles.sideContainer,
+          items: [
+            {
+              customComponent: (
+                <Text
+                  style={styles.sideText}
+                  textColor={Colors.primaryColor.background}>
+                  {formatNumber(qty)}
+                </Text>
+              ),
+            },
+            {
+              customComponent: (
+                <TextUnit
+                  style={styles.sideText}
+                  value={formatPrice(total)}
+                  unit={currencySymbol}
+                />
+              ),
+            },
+          ],
+        }}
+      />
+    );
+  }
+
+  if (typeSelect === SaleOrderLine?.typeSelect.Title) {
+    return (
+      <ObjectCard
+        style={[styles.card, style]}
+        showArrow={false}
+        touchable={false}
+        upperTexts={{
+          items: [
+            {
+              displayText: productName,
+              isTitle: true,
+            },
+            {
+              displayText: description,
+              numberOfLines: 2,
+              hideIfNull: true,
+            },
+          ],
+        }}
+      />
+    );
+  }
+
+  if (
+    typeSelect === SaleOrderLine?.typeSelect.StartOfPack ||
+    typeSelect === SaleOrderLine?.typeSelect.EndOfPack
+  ) {
+    const isStartOfPack = typeSelect === SaleOrderLine?.typeSelect.StartOfPack;
+
+    return (
+      <ObjectCard
+        style={[styles.card, style]}
+        showArrow={false}
+        touchable={false}
+        upperTexts={{
+          items: [
+            {
+              displayText: I18n.t(
+                isStartOfPack
+                  ? 'Sales_SOLineType_StartOfPack'
+                  : 'Sales_SOLineType_EndOfPack',
+              ),
+            },
+            {
+              displayText: productName,
+              isTitle: true,
+            },
+          ],
+        }}
+        sideBadges={{
+          style: styles.sideContainer,
+          items: [
+            !isStartOfPack && {
+              customComponent: (
+                <TextUnit
+                  style={styles.sideText}
+                  value={formatPrice(total)}
+                  unit={currencySymbol}
+                  defaultColor
+                />
+              ),
```

Comment:
Should be diisplay only when activated on end of pack
Also please add margin with card right border